<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnSend">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>SYS_APPASSOCIATION</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SYS_PROPERTYGET</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SYS_STARTUP</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SYS_STATUSGET</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SYS_STOP</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SYS_REBOOT</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="parseXML">
            <property name="text">
             <string>parseRequest</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="parseAll_btn">
            <property name="text">
             <string>parseResponse</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clear">
            <property name="text">
             <string>clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="configure">
              <property name="text">
               <string>configure</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="statusSetting">
              <property name="text">
               <string>statusSetting</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="configure1">
            <property name="text">
             <string>statusChange</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="statusSetting1">
            <property name="text">
             <string>statussetting1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="changeIp_btn">
            <property name="text">
             <string>changeIp</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_changeIp">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="forceConnect_btn">
            <property name="text">
             <string>forceConnect</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_udp">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
